src/query-builder/SelectQueryBuilder.ts

```diff
@@ -2318,6 +2318,10 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> implements
         }
     }
 
+    protected transformParameter(value: any, column: ColumnMetadata) {
+        return column.transformer && column.transformer.to ? column.transformer.to(value) : value;
+    }
```

Comment:
This addresses another part of the same test issue in the same commit. In the event the value is a FindOperator, we need to transform the parameters inside it, not the FindOperator itself.